sc/src/main/java/uk/ac/ebi/atlas/search/GeneSearchService.java

```diff
@@ -66,4 +69,22 @@ public GeneSearchService(GeneSearchDao geneSearchDao) {
             forkJoinPool.shutdown();
         }
     }
+
+    private  Map<String, Map<Integer, List<Integer>>> fetchPreferredKAndMinP (List<String> experimentAccessions, String geneId){
```

Comment:
Again, this method name is not quite right (I'm sorry!). Is it just returning the preferred K and the minimum probability?